homeassistant/components/deconz/__init__.py

```diff
@@ -71,8 +74,17 @@
 
     old_options = dict(config_entry.options)
 
-    new_options = {CONF_MASTER_GATEWAY: master}
-
-    options = {**old_options, **new_options}
+    options = {**old_options, CONF_MASTER_GATEWAY: master}
 
     hass.config_entries.async_update_entry(config_entry, options=options)
+
+
+async def async_add_uuid_to_config_entry(hass, config_entry):
+    """Add UUID to config entry to help discovery identify entries."""
+    gateway = get_gateway_from_config_entry(hass, config_entry)
+
+    old_config = dict(config_entry.data)
```

Comment:
This is not necessary. You can just do `**config_entry.data` below. `**` does not mutate the source object. It just copies over the values.

Reply:
Great, will fix tonight!